apps/system/js/cards_helper.js

```diff
@@ -32,8 +23,16 @@
 
       sizes.sort(function(x, y) { return y - x; });
 
-      var index = sizes[(HVGA) ? sizes.length - 1 : 0];
-      iconPath = icons[index];
+      iconPath = icons[0]; // The biggest icon available
+      for (var i = 0; i < sizes.length; i++) {
+        var size = sizes[i];
+
+        if (size < maxSize) {
```

Comment:
Should (couldnt) this be size <= maxSize? If size is 60 and there's a 60px icon available, I want to use it